ethereum/statetransition/src/testFixtures/java/tech/pegasys/teku/statetransition/datacolumns/DasCustodyStand.java

```diff
@@ -67,9 +70,13 @@ public DasCustodyStand(
     this.myNodeId = myNodeId;
     this.blockResolver = new CanonicalBlockResolverStub(spec);
     this.config = SpecConfigEip7594.required(spec.forMilestone(SpecMilestone.EIP7594).getConfig());
+    this.minCustodyPeriodSlotCalculator = MinCustodyPeriodSlotCalculator.createFromSpec(spec);
+    this.db = new DataColumnSidecarDBStub();
```

Comment:
do we need to keep `db`?

Reply:
Yes, we need it to emulate existing db data on startup, so no things are cached by the intermediate accessor classes if pushing initial data through them

Reply:
Ahh I see it's used in some tests, ok, all good, ignore this